src/oaklib/cli.py

```diff
@@ -1072,6 +1079,13 @@ def annotate(
     impl = settings.impl
     writer = _get_writer(output_type, impl, StreamingYamlWriter, datamodels.text_annotator)
     writer.output = output
+    terms_to_remove = []
+    if len(exclude_terms) == 1 and Path(exclude_terms[0]).exists():
+        with open(exclude_terms[0]) as f:
+            terms_to_remove = f.read().splitlines()
+    else:
+        terms_to_remove = list(exclude_terms)
```

Comment:
use consistent terminology. I think `token_exclusion_list` communicates what you are doing here. In ontologies people use term to mean the whole concept, not words in the label

Reply:
Done!